WordPress/gutenberg
# Interactivity API: Fix null and number strings as namespaces runtime error.

## What?

Right now, creating a namespace called `null` or an integer `24` will kill the runtime.

## Why?

We are doing a JSON.parse() function that returns `null` type or `number` type if receives `null` or `number` strings.

## How?

By checking if it is an object.

## Testing Instructions




1) Create an interactive block.
2) Use null or a number for the namespace in render.php
3) The interactive block should work.
